api/companies.py

```diff
@@ -129,14 +131,34 @@ def createCompany(json=None):
         wallet.money -= new_company_fee
         db_sess.merge(wallet)
         db_sess.commit()
-        return send_response(
+        send_response(
             event_name,
             {
                 'message': 'Success',
                 'errors': []
             }
         )
```

Comment:
@markK24 I'm not sure, that it's necessary. We have the same json lines in notification. What do you think?

Reply:
Ideally we don't get a notification about our request, but if it works then ok

Reply:
If there is no use of this message than we can delete it